editor/components/post-format/index.js

```diff
@@ -30,9 +30,10 @@ const POST_FORMATS = [
 	{ id: 'chat', caption: __( 'Chat' ) },
 ];
 
-function PostFormat( { onUpdatePostFormat, postFormat = 'standard', suggestedFormat, instanceId } ) {
+function PostFormat( { onUpdatePostFormat, postFormat = 'standard', supportedFormats, suggestedFormat, instanceId } ) {
 	const postFormatSelectorId = 'post-format-selector-' + instanceId;
-	const suggestion = find( POST_FORMATS, ( format ) => format.id === suggestedFormat );
+	const formats = POST_FORMATS.filter( ( format ) => supportedFormats.includes( format.id ) );
```

Comment:
`Array#includes` (and any other prototype-member function) is not polyfilled, and this will error in IE11.

Use [Lodash `_.includes`](https://lodash.com/docs/4.17.4#includes) instead.

See also: #746

Reply:
f7108b8d